### ISSUE TITLE
--help still shows pep8 for Usage

### ISSUE THREAD

**Issue by sgornick** [PyCQA/pycodestyle#518]
When I view the help I still see "Usage: pep8 [options] input ...".

Wouldn't pep8 be something that is no longer referenced, and pycodestyle shown instead?

```
$ pycodestyle --help
Usage: pep8 [options] input ...

Options:
  --version            show program's version number and exit
  -h, --help           show this help message and exit
[...]

$ pycodestyle --version
2.0.0
```

**Comment by IanLee1521** [comment]
Yep. Good catch! I'll probably leave that as a bug that can be fixed at the PyCon sprint on Thursday. Would be a great first contribution by someone!

**Comment by SylvainDe** [comment]
Also, wiki still has references to pep8 : https://github.com/PyCQA/pycodestyle/wiki . Not sure if better to 
have one bug open to track all these issues or one bug per issue.

**Comment by vodik** [comment]
Got it.

**Comment by IanLee1521** [comment]
Thanks for that catch @SylvainDe -- Let's split that out into a separate issue, primarily because I'm not sure that wiki content should be kept around at all.. More when I open the new issue in a minute...

**Comment by IanLee1521** [comment]
@SylvainDe -- See #531

**Comment by IanLee1521** [comment]
Thanks for the report @sgornick, this is fixed with the merging of #530.
